packages/block-editor/src/store/private-selectors.js

```diff
@@ -112,6 +112,7 @@ function getEnabledClientIdsTreeUnmemoized( state, rootClientId ) {
 export const getEnabledClientIdsTree = createRegistrySelector( ( select ) =>
 	createSelector( getEnabledClientIdsTreeUnmemoized, ( state ) => [
```

Comment:
I think some of the dependencies here can probably be removed because they're not used in the selector anymore. But I don't expect that it impacts anything.

Reply:
Good call, I'll take a look!